### ISSUE TITLE
Examples are mis-numbered

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#3138]
In the spec, the examples are numbered across section. 5.2.3 has the first example ("example 1") and the glossary item for Abbreviation has examples 2-4. This continues until the glossary item for "web content" has examples 38-42.

It seems that the ideal situation is:

- Items with individual examples would have "Example"
- items with multiple examples would have "Example 1", "Example 2", etc
- The same logic should apply to Notes. We lost numbering for multiple notes when we did WCAG 2.1

**Comment by rachaelbradley** [comment]
@awkawk  Looking at the editor's draft, the examples seem to be working as you describe the ideal. I see examples from 1-42 with each example having a number and each number having a corresponding example.  Is the concern here just that the notes should be numbered as well?

**Comment by alastc** [comment]
I think the issue with examples is that they have continuous numbering, not per-definition numbering.

I had a little dig in the XSL, but I have no idea how they are produced. One for @michael-n-cooper I'm afraid.

**Comment by awkawk** [comment]
This one really needs to be addressed before Rec.
